packages/apps/human-app/server/src/modules/jobs-discovery/jobs-discovery.service.ts

```diff
@@ -0,0 +1,47 @@
+import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
+import { lastValueFrom } from 'rxjs';
+import {
+  JobsDiscoveryParamsCommand,
+  JobsDiscoveryParamsData,
+  JobsDiscoveryResponse,
+} from './interfaces/jobs-discovery.interface';
+import { HttpService } from '@nestjs/axios';
+import { InjectMapper } from '@automapper/nestjs';
+import { Mapper } from '@automapper/core';
+
+@Injectable()
+export class JobsDiscoveryService {
+  constructor(
+    public httpService: HttpService,
+    @InjectMapper() private readonly mapper: Mapper,
+  ) {}
+
+  async processJobsDiscovery(
+    url: string,
+    jobsDiscoveryParamsCommand: JobsDiscoveryParamsCommand,
+  ): Promise<JobsDiscoveryResponse> {
+    const jobsDiscoveryParamsData = this.mapper.map(
+      jobsDiscoveryParamsCommand,
+      JobsDiscoveryParamsCommand,
+      JobsDiscoveryParamsData,
+    );
+    try {
+      const options = {
+        method: 'GET',
+        url: `${url}/jobs`,
+        params: jobsDiscoveryParamsData,
+      };
+      const response = await lastValueFrom(this.httpService.request(options));
+      return response.data;
+    } catch (error) {
```

Comment:
Didn't we agreed that this kind of error cases would be handled by exception filters?

Reply:
It's will be done in the next step -> https://github.com/orgs/blockydevs/projects/1/views/1?pane=issue&itemId=54595048